src/units.h

```diff
@@ -308,11 +308,11 @@ inline constexpr double to_liter( const volume &v )
 // Don't use in new code! Use one of the from_* functions instead.
 static constexpr volume legacy_volume_factor = from_milliliter( 250 );
 
-class mass_in_gram_tag
+class mass_in_milligram_tag
 {
 };
 
-using mass = quantity<int, mass_in_gram_tag>;
+using mass = quantity<int, mass_in_milligram_tag>;
```

Comment:
`mass` is based on an `int`, which may be just 32 bit, so the maximal value is 2147483648, which is 2,147483648 metric tons. Many generic vehicles are heavier, calculating their mass will cause a signed integer overflow, which is UB.

Consider changing it to something like `std::int_fast64_t`.

Reply:
Sure, But I have no idea on how to do it ^^"

Reply:
Change the `int` in the above declaration to `std::int_fast64_t`. Maybe add the header it's defined in ("<cstdint>"), if not already present. Change all other places that define `mass` (if there are any) accordingly.